runtime-modules/content/src/types.rs

```diff
@@ -254,14 +244,14 @@ pub type VideoUpdateParameters<T> = VideoUpdateParametersRecord<StorageAssets<T>
 pub struct VideoRecord<ChannelId, SeriesId, VideoPostId, OwnedNFT> {
     pub in_channel: ChannelId,
     pub in_series: Option<SeriesId>,
-    /// Whether the curators have censored the video or not.
-    pub is_censored: bool,
     /// enable or not comments
     pub enable_comments: bool,
     /// First post to a video works as a description
     pub video_post_id: Option<VideoPostId>,
     /// Whether nft for this video have been issued.
     pub nft_status: Option<OwnedNFT>,
+    // Whether the video has been marked as hidden by a curator
+    pub is_hidden: bool,
```

Comment:
Same point as `Channel::is_hidden`.

Reply:
Removed in https://github.com/Joystream/joystream/pull/3366/commits/3d6692e4f3234e6d2c35698ceb8a5619fb22e498